src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-tabs/GeneViewTabs.scss

```diff
@@ -7,6 +7,8 @@
   border-radius: 3px;
```

Comment:
to the `background-color` rule several lines above. I know this comes from a different PR, but `#33adff;` is our `$blue` colour; so we should use that variable instead.

Reply:
I didnt understand. I am using $blue right?

Reply:
Take a look at line 4 of this file. It says `background-color: #33adff;`. Let's change it to `background-color: $blue;`

Reply:
Aah got it. Thanks